code/datums/setup_option/backgrounds/origin_chtmant.dm

```diff
@@ -45,15 +45,15 @@
 
 /datum/category_item/setup_option/background/ethnicity/chtmantra
 	name = "Ra Caste"
-	desc = "Ra are the warriors and sentries of the hives. Numbering in the hundreds they would tower over Ru�s and even \
+	desc = "Ra are the warriors and sentries of the hives. Numbering in the hundreds they would tower over Ru�s and even \
```

Comment:
```suggestion
	desc = "Ra are the warriors and sentries of the hives. Numbering in the hundreds they would tower over Ru's and even \
```
Type

Reply:
*typo

Reply:
thank u for the aid comrade, visual studio made the ' turn to that symbol.